### PR TITLE
add XBLink 4G LTE Transmission System

### PR DESCRIPTION
**Pull request by NhanNgo2306**
add XBLink 4G LTE Transmission System

### CODE REVIEW

File: common/source/docs/common-video-landingpage.rst [hunk 1]
```diff
@@ -178,6 +178,26 @@ Unboxing video
...
+Overview
+--------
+XBLink is a Plug and Play hardware for BVLOS Flight based on Real time 4G LTE connective. Perfect fit with Cube.
+
+Features
+--------
+* Control UAV via MissionPlanner and QGroundControl
+* Platform supported: MacOS, WindowsOS, AndroidOS
+* MAVLink compatible
+* Low latency HD video streaming
```
Comment (khancyr) [inline common/source/docs/common-video-landingpage.rst:195]: Well 300-600ms isn't really low latency

Comment (Hwurzburg) [reply common/source/docs/common-video-landingpage.rst:195]:
```suggestion
* HD video streaming (300-600ms latency)
```

Comment (Hwurzburg) [reply common/source/docs/common-video-landingpage.rst:195]: if you like I can make that change and merge for you...

Comment (NhanNgo2306) [reply common/source/docs/common-video-landingpage.rst]: Thanks @khancyr  for your comments.
Thanks @Hwurzburg , I committed your suggestion.

**Comment by Hwurzburg** [comment]
squashed, tested locally (CI is broken at the moment), and will merge....thanks!!looks to be a neat system....

**Pull request merged**